.github/workflows/prepare-release.yml

```diff
@@ -0,0 +1,154 @@
+              echo "$(gh run list --branch main --json status,url --jq '[.[] | select(.status != "queued" and .status != "in_progress")][0].url' --workflow build-and-test --repo ${REPO} )"
+              exit 1
+          fi
+      # Update Contrib to use the latest in development version of Core. Run make update-otel in
+      # Contrib root directory and if it results in any changes, submit a PR. Open this PR as draft.
+      # This is to ensure that the latest core does not break contrib in any way. We’ll update it
+      # once more to the final release number later.
+      - name: Make update-otel in contrib
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          REPO: open-telemetry/opentelemetry-collector-contrib
+        run: "echo TODO" # TODO: This may require a webhook in contrib
+      - uses: actions/checkout@v3
+      - name: Setup Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: 1.19
+      # Prepare Core for release.
+      #   - Update CHANGELOG.md file, this is done via chloggen
+      #   - Run make prepare-release PREVIOUS_VERSION=1.0.0 RELEASE_CANDIDATE=1.1.0 MODSET=stable
+      #   - Run make prepare-release PREVIOUS_VERSION=0.52.0 RELEASE_CANDIDATE=0.53.0 MODSET=beta
+      - name: Prepare release for core
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          REPO: open-telemetry/opentelemetry-collector
+          CANDIDATE_BETA: ${{ inputs.candidate-beta }}
+          CANDIDATE_STABLE: ${{ inputs.candidate-stable }}
+          CURRENT_BETA: ${{ inputs.current-beta }}
+          CURRENT_STABLE: ${{ inputs.current-stable }}
+        run: |
+          make chlog-install
+          make chlog-update VERSION=${CANDIDATE_STABLE}/${CANDIDATE_BETA}
+          git config user.name "$GITHUB_ACTOR"
+          git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
+          BRANCH="prepare-release-prs/${CANDIDATE_BETA}-${CANDIDATE_STABLE}"
+          git checkout -b ${BRANCH}
+          git add --all
+          git commit -m "Changelog update ${CANDIDATE_BETA}/${CANDIDATE_STABLE}"
+          make prepare-release GH=none PREVIOUS_VERSION=${CURRENT_STABLE} RELEASE_CANDIDATE=${CANDIDATE_STABLE} MODSET=stable
+          make prepare-release GH=none PREVIOUS_VERSION=${CURRENT_BETA} RELEASE_CANDIDATE=${CANDIDATE_BETA} MODSET=beta
+          git push origin ${BRANCH}
+          gh pr create --title "[chore] Prepare release ${CANDIDATE_BETA}/${CANDIDATE_STABLE}" --body "
+          The following commands were run to prepare this release:
+          - make chlog-update VERSION=${CANDIDATE_BETA}/${CANDIDATE_STABLE}
+          - make prepare-release GH=none PREVIOUS_VERSION=${CURRENT_STABLE} RELEASE_CANDIDATE=${CANDIDATE_STABLE} MODSET=stable
+          - make prepare-release GH=none PREVIOUS_VERSION=${CURRENT_BETA} RELEASE_CANDIDATE=${CANDIDATE_BETA} MODSET=beta
+          "
+
+# Still TODO manually:
```

Comment:
Is this pending because of technical challenges? Or because it just wasn't done yet?